homeassistant/components/tasmota/sensor.py

```diff
@@ -108,14 +108,15 @@
     SENSOR_PRESSUREATSEALEVEL: {DEVICE_CLASS: DEVICE_CLASS_PRESSURE},
     SENSOR_PROXIMITY: {ICON: "mdi:ruler"},
     SENSOR_REACTIVE_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
+    SENSOR_STATUS_RESTART: {ICON: "mdi:information-outline"},
     SENSOR_STATUS_SIGNAL: {DEVICE_CLASS: DEVICE_CLASS_SIGNAL_STRENGTH},
     SENSOR_STATUS_RSSI: {ICON: "mdi:access-point"},
+    SENSOR_STATUS_UPTIME: {ICON: "mdi:progress-clock"},
```

Comment:
is there a reason this changed from DEVICE_CLASS_TIMESTAMP? there may be some differences in frontend formatting without such.

Reply:
Yes, it's because the uptime is not a timestamp, it measures elapsed time.

Reply:
Uptime has its own format: `[n days]T[hh:mm:ss]`. 
```py
{"Uptime":"3T13:38:27"}
```
To have it working as DEVICE_CLASS_TIMESTAMP a conversion is needed by our side and frankly doesn't worth the effort since is already sowing a clear information.

Reply:
while it may seem not worth it, this is going to  create a lot of state changes which is why using timestamp is preferrred as you would only set the last boot time

Reply:
Hmm, that's a good point. Storing the timestamp of last restart would be enough, but it's maybe not so user friendly for display?

Reply:
Our standard is to use device class timestamp for these cases. Please convert the time to an absolute time if needed.

Reply:
Thanks Martin, we'll get that sorted in next hatasmota version.

Reply:
Corrected in #43013